packages/ts/src/core/container/index.ts

```diff
@@ -62,15 +68,24 @@ export class ContainerCore {
     }
 
     // Apply the `aria-label` attribute
-    select(this._container)
-      .attr('aria-label', config.ariaLabel)
+    let description = ''
+    if (config.ariaLabel) {
+      description = config.ariaLabel
+    } else if (config.ariaAutoLabel) {
+      description = this._getAriaDescription().trim()
+    }
+    if (description) {
```

Comment:
If `config.ariaLabel` and `config.ariaAutoLabel` were set to `false` in a config update, the chart will still keep its old description.

And I would like to make this block look a little cleaner by moving the `description` logic inside `.attr('aria-label', ...)` or into a separate method.

```ts
    let description = ''
    if (config.ariaLabel) {
      description = config.ariaLabel
    } else if (config.ariaAutoLabel) {
      description = this._getAriaDescription().trim()
    }

    if (description) {
      select(this._container)
        .attr('aria-label', description)
        .attr('tabindex', '0')
```